octopoes/tests/robot/05_bits.robot

```diff
@@ -12,8 +14,8 @@ Bit With Scan Level 1
     Object List Should Not Contain    Finding|Hostname|internet|example.com|KAT-NXDOMAIN
     Declare Scan Profile    Hostname|internet|example.com    1
     Await Sync
-    Object List Should Contain    KATFindingType|KAT-NXDOMAIN
-    Object List Should Contain    Finding|Hostname|internet|example.com|KAT-NXDOMAIN
+    Wait Until Keyword Succeeds    60s    1s    Object List Should Contain    KATFindingType|KAT-NXDOMAIN
+    Wait Until Keyword Succeeds    60s    1s    Object List Should Contain    Finding|Hostname|internet|example.com|KAT-NXDOMAIN
```

Comment:
Does this still fail the test if it isn't there after 60 seconds? The robot framework documentation isn't very clear but I think they mean to say that it doesn't.

Reply:
Yes, I believe that is the gist of it; see [here](http://robotframework.org/robotframework/2.6.1/libraries/BuiltIn.html).